WordPress/src/androidTest/java/org/wordpress/android/ui/screenshots/WPScreenshotTest.java

```diff
@@ -59,7 +59,6 @@ public void wPScreenshotTest() {
         Screengrab.setDefaultScreenshotStrategy(new UiAutomatorScreenshotStrategy());
 
         wpLogin();
-        sleep();
```

Comment:
@jkmassel I removed this because I think I had included it in an earlier PR and shouldn't be necessary anymore